### DISCUSSION TITLE
pybamm.plot_voltage_components() has different results on Linux and Windows

### DISCUSSION THREAD

**Discussion by zhengq01** [pybamm-team/PyBaMM#2071] [Q&A]
Hi, there! I am a newer of PyBaMM and I am trying to run the code in notebooks/Getting_started. When I run Tutorial 3 - Basic plotting in Windows, I find that the results of pybamm.plot_voltage_components() is not normal as shown in the figure below. 
![voltage](https://user-images.githubusercontent.com/16679647/169738207-cc020c24-1a5b-4a09-b713-c8271182ca9e.jpg)
One can see that the legend is not right (i.e., voltage should be described with black dashed line), however, when I transfer to Linux, it is right. I wonder whether someone meets the similar problem? My python version is 3.7.1 and PyBaMM version is 22.4. Thanks a lot!

**Comment by TomTranter** [comment]
Seems this was fixed in #2061 you might be running slightly different versions of develop branch I guess

**Reply by zhengq01** [reply]
It works! Thanks a lot!
